test/specs/miscellaneous.e2e.ts

```diff
@@ -44,7 +50,7 @@ describe('Miscellaneous', async () => {
     const workbench = await utilities.getWorkbench();
     const commandName =
       EnvironmentSettings.getInstance().vscodeVersion === 'stable' ||
-      semver.gte(EnvironmentSettings.getInstance().vscodeVersion, '1.92.0')
+        semver.gte(EnvironmentSettings.getInstance().vscodeVersion, '1.92.0')
```

Comment:
The logic here failed locally when default _vscodeVersion is still stable and the running version of VSCode is 1.91.0. @daphne-sfdc

Reply:
Hmmm It works on my computer though. Nvm